R/tidyeval.R

```diff
@@ -270,6 +270,8 @@ dt_squash_call <- function(x, env, data, j = TRUE) {
     x[[1]] <- expr(rleid)
     x[-1] <- lapply(x[-1], dt_squash, env, data, j = j)
     x
+  } else if (is_call(x, c("$", "[[")) && !is_mask_pronoun(x)) {
```

Comment:
I don't think `&& !is_mask_pronoun(x)` has an effect here, since this is an `else if` after `if (is_mask_pronoun(x))`. Looks good either way, that's just a code style preference (maybe you wanted it to be more explicit?)

Reply:
Ah yep good catch. I'll drop that.